tests/apis.js

```diff
@@ -478,28 +685,31 @@ test('API: ofType with Follow', t => {
   });
 });
 
-test('API: One', t => {
-  const item = Math.round(Math.random() * (content.length - 1));
-  let expected = cloneDeep(content[item]);
+test.serial.skip('API: One', t => {
```

Comment:
In addition, this doesn't appear to actually test to make sure that we've tested for references.